packages/types/src/interfaces.ts

```diff
@@ -1,10 +1,8 @@
 // Copyright 2020-2022 OnFinality Limited authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import {AnyTuple, Codec} from '@polkadot/types-codec/types';
-import {GenericExtrinsic} from '@polkadot/types/extrinsic';
-import {EventRecord, SignedBlock} from '@polkadot/types/interfaces';
-import {IEvent} from '@polkadot/types/types';
+import BN from 'bn.js';
+import {Chunk} from 'near-api-js/lib/providers/provider';
```

Comment:
It seems we can import all the type (BlockHeader etc) from here. It would be preferable not to redefine things.